### ISSUE TITLE
Tapo C100 v5 supported?

### ISSUE THREAD

**Issue by ikout** [themactep/thingino-firmware#378]
Hello, I got a new Tapo C100 but the only available was a v5 EU. Is this device going to work?
When I look inside I see that the main CPU is a T23 and the Wifi chip seem to be different..

Thingino site says for C100: "T31L, SC2336, RTL8188FTV, 8MB"
The device I have is based on the T23 CPU and for WiFi it seems to have a WQ9001 chip...

Any ideas if this device will be possible to be flashed with Thingino? And which settings???

**Comment by gtxaspec** [comment]
Cpu is supported, wifi not yet... We are missing the driver sources.

**Comment by ikout** [comment]
So can't flash it then.... :( 
That's unfortunate, and I really wanted to test Thingino...

**Comment by themactep** [comment]
Buy a supported camera from the list at http://thingino.com/
Say, Cinnado D1 is available in EU.

**Comment by themactep** [comment]
Guys, you want new hardware supported - send a specimen to developers. We cannot work on models that we do not have on hands.

**Comment by gtxaspec** [comment]
@ikout initial support has been added for the WQ9001 wifi chip.  Would you please confirm that your model has this chip?

**Comment by ikout** [comment]
Hi @gtxaspec, 

I believe that that's the one we are talking about, so yes.

<img width="956" alt="image" src="https://github.com/user-attachments/assets/31888909-e90c-4f8e-94e0-22017a0b0135" />

**Comment by gtxaspec** [comment]
@ikout do you have a dump of the stock firmware?

**Comment by ikout** [comment]
No, I've not tried anything yet since there was no support on this device in general. 

I think I've seen 4 pads that maybe for serial or USB. I don't mind to brick temporarily, I got this one to use Thingino anyway (if possible)

**Comment by gtxaspec** [comment]
see:
https://github.com/themactep/thingino-firmware/commit/5d0e8d748891ec39d673722e96981a0c18333748

**Comment by gtxaspec** [comment]
added
